Xamarin.Essentials/HapticFeedback/HapticFeedback.uwp.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Diagnostics;
+using System.Threading.Tasks;
+using Windows.Devices.Haptics;
+
+namespace Xamarin.Essentials
+{
+    public static partial class HapticFeedback
+    {
+        internal static bool IsSupported => true;
```

Comment:
Maybe do a similar ApiInformation check like with the Vibration... I believe this API is only available in newer version of Windows. Might explode on older versions.

Reply:
Yes, I will add a check.